index.d.ts

```diff
@@ -0,0 +1,85 @@
+type SearchClient = object;
+
+interface TypesenseNode {
+  host: string;
+  port: string;
+  protocol: string;
```

Comment:
And Protocol should be always a choice between HTTP / HTTPS. You can make a enum for example something like `supported-protocols.enum.ts` where there will be all the protocol listed and allows only protocols which are listed there. :) People can quickly see in intelli sense what protocols are supported!

Reply:
Fixed that !